data/sql_test.go

```diff
@@ -0,0 +1,68 @@
+package data_test
+
+import (
+	"database/sql"
+	"reflect"
+	"strconv"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+func ExampleSQLStringConverter() {
```

Comment:
Nit, but maybe considered putting sql stuff in separate sub-package of data.

Reply:
Been on my mind. The replacer might become more general as well based on some stuff Ryan brought up to me on other plugin work.

So will keep this option in mind.